src/NumericalAlgorithms/Spectral/Mesh.hpp

```diff
@@ -48,6 +49,39 @@ class Mesh {
  public:
   static constexpr size_t dim = Dim;
 
+  struct Extents {
+    using type = size_t;
+    static constexpr Options::String help = {
+        "The number of collocation points per dimension"};
```

Comment:
Question: Do we want to support different extents in different dimensions?

Reply:
@nilsleiffischer suggested to wait until #2625 is finished and it can be added later.

Reply:
OK.  Sounds good.